src/PayU/Alu/HTTPClient.php

```diff
@@ -1,66 +1,15 @@
 <?php
 
-namespace PayU\Alu;
 
-use PayU\Alu\Exceptions\ClientException;
-use PayU\Alu\Exceptions\ConnectionException;
+namespace PayU\Alu;
 
-class HTTPClient
+/**
+ * Class HTTPClient
+ * @package PayU\Alu
+ * @deprecated Will be removed on a further library version.
+ * Depending on PaymentsApi version, should use AluV3/Services/HTTPClient or PaymentsV4/Services/HTTPClient
+ */
+class HTTPClient extends \PayU\PaymentsApi\AluV3\Services\HTTPClient
```

Comment:
Instead of extending PayU\PaymentsApi\AluV3\Services\HTTPClient, this new class could use AluV3\Services\HTTPClient as a dependancy. Same for HashService. If we'd do this, the new HTTPClient and HashService classes from AluV3 namespace could be declared final as well. Right now they're the only new classes that are not final and just because of this inheritance.